scipy/stats/_multivariate.py

```diff
@@ -791,26 +791,28 @@ def fit(self, x):
 
         Parameters
         ----------
-        x : data the distribution is fitted to. The last axis determines the
-            dimension of the fitted distribution.
+        x : ndarray (m, n)
+            Data the distribution is fitted to. Must be two dimensional.
```

Comment:
We use "dimension" in the next sentence to refer to `n` rather than the `ndim` of the input. Might want to consider "Must have exactly two axes."